### DISCUSSION TITLE
Error 4214

### DISCUSSION THREAD

**Discussion by metalclaw22** [Melledy/Grasscutter#1877] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/84073557/196834919-d58c69f5-cec4-436e-88fa-7b76261a1c32.png)

i cant enter the game, i have followed all the steps but nothing

**Comment by ChasonJiang** [comment, marked as answer]
you need a patch file that called "UserAssembly.dll" in current version, and you could be found in Discord

**Reply by metalclaw22** [reply]
Okay thank you
